lib/Common/CommonDefines.h

```diff
@@ -186,9 +186,9 @@
 #ifndef ENABLE_VALGRIND
 #define ENABLE_CONCURRENT_GC 1
 #ifdef _WIN32
-#define ENABLE_ALLOCATIONS_DURING_CONCURRENT_SWEEP 0 // Only takes effect when ENABLE_CONCURRENT_GC is enabled.
+#define ENABLE_ALLOCATIONS_DURING_CONCURRENT_SWEEP 1 // Only takes effect when ENABLE_CONCURRENT_GC is enabled.
 #else
-#define ENABLE_ALLOCATIONS_DURING_CONCURRENT_SWEEP_USE_SLIST 1 // Use Interlocked SLIST for allocableHeapBlockList
+#define ENABLE_ALLOCATIONS_DURING_CONCURRENT_SWEEP 0 // Needs ENABLE_CONCURRENT_GC to be enabled for this to be enabled.
```

Comment:
Is this comment correct? this looks like `ENABLE_CONCURRENT_GC` is defined to 1 just above, it's just not on win32?